src/SegmentedMsg.chpl

```diff
@@ -108,6 +114,11 @@ module SegmentedMsg {
         st.addEntry(neName, new shared SymEntry(segArr.getNonEmpty()));
         repMsg = "created " + st.attrib(neName) + "+" + segArr.getNonEmptyCount():string;
       }
+      when (DType.BigInt){
+        var segArr = getSegArray(name, st, bigint);
+        st.addEntry(neName, new shared SymEntry(segArr.getNonEmpty()));
+        repMsg = "created " + st.attrib(neName) + "+" + segArr.getNonEmptyCount():string;
+      }
```

Comment:
This is another spot where it matches the existing code. However, I think we could do this without creating a new `SymEntry`. Could we add an issue to update this to look at doing something like
```chapel
var segArr = getSegArray(name, st, type);
repMsg = "created " + st.attrib(name) + "+" + segArr.getNonEmptyCount():string;
```